cmd/bbolt/main.go

```diff
@@ -1180,17 +1182,25 @@ func (cmd *KeysCommand) Run(args ...string) error {
 
 		// Iterate over each key.
 		return b.ForEach(func(key, _ []byte) error {
-			fmt.Fprintln(cmd.Stdout, string(key))
+			writeBytes(cmd.Stdout, key, *optionsFormat)
+			if *optionsFormat=="bytes" {
+				fmt.Fprintln(cmd.Stdout)
+			}
 			return nil
 		})
 	})
 }
 
 // Usage returns the help message.
+// TODO: Use https://pkg.go.dev/flag#FlagSet.PrintDefaults to print supported
+//       flags.
 func (cmd *KeysCommand) Usage() string {
 	return strings.TrimLeft(`
 usage: bolt keys PATH BUCKET
 
+	--format
```

Comment:
nit, lgtm but probably a good idea to add a line above as `options:` or something like that to show available options. Thanks!